key.go

```diff
@@ -428,23 +449,39 @@ func LoadPublicKeyFromDER(der_block []byte) (PublicKey, error) {
 	return p, nil
 }
 
-// GenerateRSAKey generates a new RSA private key with an exponent of 3.
+// GenerateRSAKey generates a new RSA private key with an exponent of 65537.
 func GenerateRSAKey(bits int) (PrivateKey, error) {
-	return GenerateRSAKeyWithExponent(bits, 3)
+	return GenerateRSAKeyWithExponent(bits, 65537)
```

Comment:
why not 17? ;-)
could we have 65537 as a KeyExponent const somewhere?

Reply:
there is acctually a #define for it: `RSA_F4`
I got confused, the usual code that creates the key uses:
```C
rc=BN_set_word(bn, RSA_F4)
```
Either RSA_F4 or ow own const would be nice here, perhaps?